pkgs/development/python-modules/neurokit2/default.nix

```diff
@@ -0,0 +1,56 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3Packages,
+  buildPythonPackage,
+}:
+
+buildPythonPackage rec {
+  pname = "neurokit2";
+  version = "0.2.10";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "neuropsychology";
+    repo = "NeuroKit";
+    tag = "v${version}";
+    hash = "sha256-e/B1JvO6uYZ6iVskFvxZLSSXi0cPep9bBZ0JXZTVS28=";
+  };
+
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace-fail '"pytest-runner"' '"pytest"'
+  '';
+
+  build-system = with python3Packages; [
+    setuptools
+    wheel
+    pytest_7
+  ];
+
+  dependencies = with python3Packages; [
+    scipy
+    scikit-learn
+    pandas
+    matplotlib
+    requests
+    cvxopt
+    biosppy
+
+    # Test Dependencies
```

Comment:
Same comments as for biosppy regarding `python3Packages`, test dependencies/`nativeCheckInputs`.